.github/workflows/images.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     uses: ./.github/workflows/image-builder.yml
     with:
       group: workspaces
-      images: prairielearn/workspace-desktop,prairielearn/workspace-jupyterlab-python,prairielearn/workspace-rstudio,prairielearn/workspace-xtermjs,prairielearn/workspace-vscode-base,prairielearn/workspace-vscode-cpp,prairielearn/workspace-vscode-python
+      images: prairielearn/workspace-desktop,prairielearn/workspace-jupyterlab-python,prairielearn/workspace-rstudio,prairielearn/workspace-xtermjs,prairielearn/workspace-vscode-base,prairielearn/workspace-vscode-cpp,prairielearn/workspace-vscode-python,prairielearn/workspace-vscode-java
```

Comment:
Nit, here and below and in other files: I'd been loosely maintaining alphabetical order (though I now see that the xtermjs workspace isn't ordered).